crates/modelardb_server/src/storage/mod.rs

```diff
@@ -360,15 +343,15 @@ impl StorageEngine {
         Ok(())
     }
 
-    /// Retrieve the compressed sorted files that correspond to the column at `column_index` in the
-    /// table with `table_name` within the given range of time and value. If some compressed data
-    /// that belongs to `column_index` in `table_name` is still in memory, save it to disk first. If
-    /// no files belong to the column at `column_index` for the table with `table_name` an empty
-    /// [`Vec`] is returned, while a [`DataRetrievalError`](ModelarDbError::DataRetrievalError) is
-    /// returned if:
+    // TODO: Pass the metadata manager that the files should be retrieved from as an argument.
```

Comment:
Unclear why there is a TODO here as the metadata manager does not seem to be used.

Reply:
Removed the TODO since it was just so I would remember what to change when adding querying for the manager.